bin/upload-docs

```diff
@@ -2,22 +2,24 @@
 set -ef -o pipefail
 
 bucket="chain-staging.chain.com"
-if [ "$1" = "prod" ];
+if [ "$2" = "prod" ];
```

Comment:
Let's document the updated command line arguments in https://github.com/chain/chain/blob/main/docs/README.md